common/ajax-fetch/ajax-fetch-en.html

```diff
@@ -0,0 +1,69 @@
+---
+{
+	"title": "Ajax Fetch",
+	"language": "en",
+	"altLangPage": "ajax-fetch-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-08-25"
+}
+---
+<div class="wb-prettify all-pre"></div>
+<h2>Successful Ajax Fetch</h2>
+<div data-ajax-replace="ajax/data-ajax-extra-en.html">
+	<section>
+		<h3>Text to be replaced</h3>
+		<p>This text will be replaced.</p>
+	</section>
+</div>
+
+<h2>Failed Ajax Fetch</h2>
+<div data-ajax-replace="failedUrl">
+	<section>
+		<h3>Text to be replaced</h3>
+		<p>This text will <strong>not</strong> be replaced.</p>
+	</section>
+</div>
+
+<div class="row">
```

Comment:
We would just need an heading level 2

```suggestion
<h2>Code</h2>
<div class="row">
```